auth/src/main/java/com/firebase/ui/auth/data/model/FlowParameters.java

```diff
@@ -151,4 +152,10 @@ public boolean isPrivacyPolicyUrlProvided() {
     public boolean isAnonymousUpgradeEnabled() {
         return enableAnonymousUpgrade;
     }
+
+    public Bundle toBundle() {
+        Bundle bundle = new Bundle();
+        bundle.putParcelable(ExtraConstants.FLOW_PARAMS, this);
+        return bundle;
+    }
```

Comment:
Nit: could you move this to right below the the `fromIntent` method?

Reply:
Of course.